QModManager/API/SMLHelper/Handlers/BioReactorHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace QModManager.API.SMLHelper.Handlers
+{
+    using Patchers;
+    using Interfaces;
+
+    /// <summary>
+    /// A handler class for managing Bio Reactor fuels
+    /// </summary>
+    public class BioReactorHandler : IBioReactorHandler
```

Comment:
Custom build events for automatically converting these classes to support dependency injection?